src/router/index.ts

```diff
@@ -1,13 +1,18 @@
 import Vue from "vue";
 import VueRouter from "vue-router";
 import LicitatiiView from "@/views/LicitatiiView.vue";
+import TestDeckComponent from "@/components/TestDeckComponent.vue";
```

Comment:
TestDeckComponent should be DeckComponent?

Reply:
All test components will be deleted. They are only a placeholder used to demonstrate stuff. In this case, a mixin pattern which is quite important since there is not a lot of documentation on it.